packages/netlify-cms-backend-github/src/API.ts

```diff
@@ -12,6 +12,22 @@ import {
   resolvePromiseProperties,
   ResponseParser,
 } from 'netlify-cms-lib-util';
+import {
```

Comment:
We the '@octokit/rest' as a dev dependency due to the cypress GitHub tests.
Found it very useful to import the REST API type definitions.